pinot-common/src/main/java/com/linkedin/pinot/common/data/StarTreeIndexSpec.java

```diff
@@ -36,6 +36,10 @@
   private Set<String> _skipMaterializationForDimensions;
   private int skipMaterializationCardinalityThreshold = DEFAULT_SKIP_MATERIALIZATION_CARDINALITY_THRESHOLD;
 
+  private boolean enableHllColumn = false;
+  private String hllColumnName;
+  private int hllLog2m;
```

Comment:
what is this value ? provide a short comment and also javadoc for corresponding getter